tools/clang/test/HLSLFileCheck/d3dreflect/rdat_mintarget/sm69_mesh_node_deriv.hlsl

```diff
@@ -0,0 +1,202 @@
+// RUN: %dxilver 1.8 | %dxc -T lib_6_9 %s | %D3DReflect %s | %FileCheck %s -check-prefixes=RDAT
+
+// Ensure that categories of deriv ops are allowed for node shaders.
+// Ensure that the OptFeatureInfo_UsesDerivatives flag is set as well.
+
+// RDAT: FunctionTable[{{.*}}] = {
+
+Texture2D<float4> T2D : register(t0, space0);
+SamplerState Samp : register(s0, space0);
+RWByteAddressBuffer BAB : register(u1, space0);
+
+///////////////////////////////////////////////////////////////////////////////
+// Category: derivatives ddx/ddy/ddx_coarse/ddy_coarse/ddx_fine/ddy_fine
+
+// RDAT-LABEL: UnmangledName: "node_deriv"
+// RDAT: FeatureInfo1: 0
```

Comment:
This should be reporting the `DerivativesInMeshAndAmpShaders` feature flag requirement.  In order to do that, you need to update `AdjustMinimumShaderModelAndFlags` in `DxilModule.cpp` under the `if (flags.GetUsesDerivatives()) {` block to add a node case, and check the launch mode, then set `flags.SetDerivativesInMeshAndAmpShaders(true);`.